packages/config/config/devices/0x021f/dmmz1.json

```
@@ -14,6 +14,28 @@
 		"max": "255.255"
 	},
 	"supportsZWavePlus": true,
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Basic_Set",
+			"description": "The Mouser sends a Basic Set command to Association Group 2 (or the Control Group) to directly trigger devices (like a light, chime, etc.) when it is tripped.",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "Notification_Report",
+			"description": "Group 3 supports up to 5 members and the Mouser sends it a NOTIFICATION_REPORT when tripped, reset, armed, and disarmed.",
```

```suggestion
			"description": "Send a Notification Report when tripped, reset, armed, and disarmed.",
```
like above - make it short and leave out unnecessary information